components/AfricaMap.vue

```diff
@@ -21,112 +14,114 @@ export default {
       title: 'Highmaps (Africa) basic demo',
       mapChart: {
         // It's important for mapName to match exactly
-        // chartOptions.chart.map below 
+        // chartOptions.chart.map below
         // (since that's where the library renders to)
         mapName: 'africa',
         mapData: AfricaMapData,
         // mapData: '/africa.geo.json' // Also works (if this is in static folder)
       },
-      watchers: ['options.title']
+      watchers: ['options.title'],
     }
   },
   computed: {
     chartOptions() {
       return {
         chart: {
-          map: 'africa'
+          map: 'africa',
```

Comment:
I think this is fine for now, but I think eventually it'd make for much cleaner code if the chartOptions were stored somewhere else (in their own json), and then imported as needed, tweaking only the props of interest, such as the title text. But for now, this is fine.

Reply:
that could work indeed and make things a bit simpler